File: programming-basics-javascript/01a-first-steps-in-coding-lab/09-yard-greening.js
function yardGreening(input) {
  let squareMeters = Number(input[0]);
  priceStart = squareMeters * 7.61;
  priceDiscount = priceStart * 0.18;
  priceFinal = priceStart - priceDiscount;
  console.log(`The final price is: ${priceFinal} lv.`);
  console.log(`The discount is: ${priceDiscount} lv.`);
}

File: b1-programming-basics-python/01a-first-steps-in-coding-lab/09_yard_greening.py
greening_area = float(input())

greening_area_cost = greening_area * 7.61
greening_area_discount = greening_area_cost * 0.18
greening_area_final_cost = greening_area_cost - greening_area_discount

print(f"The final price is: {greening_area_final_cost} lv.")
print(f"The discount is: {greening_area_discount} lv.")
